templates/slotViz.html

```diff
@@ -416,40 +403,46 @@
         if (!slotVizCountdown) {
           slotVizCountdown = document.createElement("div")
           slotVizCountdown.className = "mb-3"
-          $("#" + selector).prepend(slotVizCountdown)
+          $(selector).prepend(slotVizCountdown)
         }
         const slots = Math.ceil((hfTs - now) / 1000 / secondsPerSlot)
-        slotVizCountdown.innerHTML = `<b>Hardfork in:</b> ${getRelativeTime(luxon.DateTime.fromMillis(hfTs))} | ${slots} slots`
+        slotVizCountdown.innerHTML = `<b>${hardforkName} Hardfork in:</b> ${getRelativeTime(luxon.DateTime.fromMillis(hfTs))} | ${slots} slots`
```

Comment:
This always states "slots" (i.e. plural), even when there is only 1 slot left. This is weird as it works properly for seconds.

Reply:
slot/slots will now be displayed dynamically